### DISCUSSION TITLE
Congratulations and Thank You

### DISCUSSION THREAD

**Discussion by solarpunked** [lightningrodlabs/acorn#58] [Ideas]
We've started using Acorn on a project and would like to congratulate the team on a very strong alpha release. Thanks to everyone who has and will contribute.

Had a few things that weren't bugs, but some initial ux/ui feedback:

- The icon on the goals that opens up the edit dialog looks like an icon to be able to the move the goal. From my perspective of ux it would be good to have the 1 edit dialog AND also be able to move goals to other parts of the screen real estate. 

- At the moment the way to move a goal to be a child of another goal is to first unlink the goals and then draw a new line from parent to child. It would be great to just draw a line line either parent to child or child to parent and the goal moves.

That's all that comes to mind after initial use. 

Really impressed.

Thanks again.

**Comment by Connoropolous** [comment]
Hey @solarpunked ! 

Thanks for this. We are glad you found this discussion space to chime in! You are the first, so you get a 🏆 . 

On your points: 
"The icon on the goals that opens up the edit dialog looks like an icon to be able to the move the goal. From my perspective of ux it would be good to have the 1 edit dialog AND also be able to move goals to other parts of the screen real estate."
Great feedback, will put in the UX queue for discussion with @pegahvaezi 

"At the moment the way to move a goal to be a child of another goal is to first unlink the goals and then draw a new line from parent to child. It would be great to just draw a line line either parent to child or child to parent and the goal moves."
Incidentally, this IS actually a bug. The behaviour worked as you described, then an update seems to have broke it. It will be fixed in the next release. 

Please continue to share thoughts with us. 
Connor

**Reply by solarpunked** [reply]
Thanks Connor,

It's pleasing that the feedback is welcomed. We're currently using Trello for a few projects. The way things can easily be moved from one place to another is one of the big factors of why we used Trello at all. So we can easily make a visual hierarchy based on what we think is important. That's the driving force behind both of the points mentioned.

Whilst we're speaking about Trello another great thing we like about it, is clicking on the item in the board goes to the edit dialog for that item and the little pencil icon on the item allows you to quickly edit the name of the item without going into the edit dialog. That would be worth taking note of when you are discussing ux as well.

**Comment by TechSupport26** [comment]
Updating to version 7.0.1 was very smooth. Thanks for the quick work.

**Reply by Connoropolous** [reply]
Great to hear! We want using Acorn to be as stable and as enjoyable as possible.

**Comment by TechSupport26** [comment]
I just noticed the "breadcrums" in the top left when you are in a card. This is awesome for navigation. I feel that this is very underrated 😁.

**Reply by Connoropolous** [reply]
Nice. Glad you discovered them. We have to figure out how to make them work for multiple parents

**Reply by TechSupport26** [reply]
Perhaps when you hover the cursor over it will expand and show other parent cards in a sideways tree fashion like so.
![image](https://github.com/lightningrodlabs/acorn/assets/65631811/b083a9d9-e77a-4d6d-9efa-203b248c8c5b)

It could look like this
![image](https://github.com/lightningrodlabs/acorn/assets/65631811/36db0aa3-ef28-4847-91c9-f5e63ac95b7a)

And the tree format is this
![image](https://github.com/lightningrodlabs/acorn/assets/65631811/1389d580-8708-467c-9785-742daf15d9e8)

Feel free to make this into an issue. 🙂
